The job **`mingw-check`** failed! Check out the build log: [(web)](https://github.com/rust-lang/rust/actions/runs/5936262088/job/16096322458) [(plain)](https://github.com/rust-lang/rust/commit/0021b0f4ec608cd411f12d08a98767597ed0c779/checks/16096322458/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
    Checking rustc_infer v0.0.0 (/checkout/compiler/rustc_infer)
error: unclosed HTML tag `p`
  --> compiler/rustc_index/src/lib.rs:38:9
   |
38 | /// </p><p>
   |
   = note: `-D rustdoc::invalid-html-tags` implied by `-D warnings`

error: could not document `rustc_index`
error: could not document `rustc_index`

Caused by:
  process didn't exit successfully: `/checkout/obj/build/bootstrap/debug/rustdoc --edition=2021 --crate-type lib --crate-name rustc_index compiler/rustc_index/src/lib.rs --target x86_64-unknown-linux-gnu -o /checkout/obj/build/x86_64-unknown-linux-gnu/stage0-rustc/x86_64-unknown-linux-gnu/doc --cfg 'feature="default"' --cfg 'feature="nightly"' --cfg 'feature="rustc_macros"' --cfg 'feature="rustc_serialize"' -Zunstable-options --check-cfg 'values(feature, "default", "nightly", "rustc_macros", "rustc_serialize")' --check-cfg 'names()' --check-cfg 'values()' --error-format=json --json=diagnostic-rendered-ansi,artifacts,future-incompat -C metadata=5417645f37677cba -L dependency=/checkout/obj/build/x86_64-unknown-linux-gnu/stage0-rustc/x86_64-unknown-linux-gnu/release/deps -L dependency=/checkout/obj/build/x86_64-unknown-linux-gnu/stage0-rustc/release/deps --extern arrayvec=/checkout/obj/build/x86_64-unknown-linux-gnu/stage0-rustc/x86_64-unknown-linux-gnu/release/deps/libarrayvec-f143db7d85461433.rmeta --extern rustc_macros=/checkout/obj/build/x86_64-unknown-linux-gnu/stage0-rustc/release/deps/librustc_macros-fb99f5ded5a1b981.so --extern rustc_serialize=/checkout/obj/build/x86_64-unknown-linux-gnu/stage0-rustc/x86_64-unknown-linux-gnu/release/deps/librustc_serialize-502fc980623ac540.rmeta --extern smallvec=/checkout/obj/build/x86_64-unknown-linux-gnu/stage0-rustc/x86_64-unknown-linux-gnu/release/deps/libsmallvec-720d9ea4f4342760.rmeta --extern-html-root-url 'arrayvec=https://docs.rs/arrayvec/0.7.4/' --extern-html-root-url 'smallvec=https://docs.rs/smallvec/1.11.0/' -Zunstable-options --cfg=bootstrap --cfg=windows_raw_dylib -Csymbol-mangling-version=v0 -Zunstable-options '--check-cfg=values(bootstrap)' '--check-cfg=values(parallel_compiler)' '--check-cfg=values(no_btreemap_remove_entry)' '--check-cfg=values(crossbeam_loom)' '--check-cfg=values(span_locations)' '--check-cfg=values(rustix_use_libc)' '--check-cfg=values(emulate_second_only_system)' '--check-cfg=values(windows_raw_dylib)' --document-private-items --document-hidden-items -Dwarnings '-Wrustdoc::invalid_codeblock_attributes' --crate-version '1.74.0-nightly (7b8f296d0 2023-08-22)' --document-private-items '-Arustdoc::private-intra-doc-links' --enable-index-page -Zunstable-options -Znormalize-docs --show-type-layout --generate-link-to-definition '-Zcrate-attr=warn(rust_2018_idioms)' --extern-html-root-url 'ena=https://docs.rs/ena/latest/'` (exit status: 1)
Build completed unsuccessfully in 0:00:37
  local time: Tue Aug 22 08:34:20 UTC 2023
  network time: Tue, 22 Aug 2023 08:34:20 GMT
##[error]Process completed with exit code 1.
```

</details>

--- a/compiler/rustc_index/src/lib.rs
+++ b/compiler/rustc_index/src/lib.rs
@@ -29,13 +29,18 @@ pub use {idx::Idx, slice::IndexSlice, vec::IndexVec};
 pub use rustc_macros::newtype_index;
 
 /// Type size assertion. The first argument is a type and the second argument is its expected size.
-/// Note to the reader: Emitting hard errors from size assertions like this is generally not
+///
+/// <div class="warning">
+///
+/// Emitting hard errors from size assertions like this is generally not
 /// recommended, especially in libraries, because they can cause build failures if the layout
 /// algorithm or dependencies change. Here in rustc we control the toolchain and layout algorithm,
 /// so the former is not a problem. For the latter we have a lockfile as rustc is an application and
 /// precompiled library.
 ///
 /// Short version: Don't copy this macro into your own code. Use a `#[test]` instead.
+///
+/// </div>
 #[macro_export]
 macro_rules! static_assert_size {
     ($ty:ty, $size:expr) => {

